### DISCUSSION TITLE
After configuring for webrtc how to verify streams are using webrtc on lan and also through firewalled wan?

### DISCUSSION THREAD

**Discussion by chrislawso** [blakeblackshear/frigate#15479] [Ask A Question]
In version 0.14.1 and any newer versions, after completing webrtc configuration https://docs.frigate.video/configuration/live/#webrtc-extra-configuration
and defining the audio codec to transcode and adjusting to ` #audio=opus` how can it be verified which, or if any, cameras are streaming using webrtc? 

In older frigate versions each video had some button to manually select and change the desired stream ie mse, webrtc etc. In the new 0.14.x and newer versions I don't see any webrtc information identified anywhere. The only mention I can find of webrtc is in systemlogs > Go2rtc:
```
Type
info
Timestamp
2024-12-12 12:12:12
Tag
webrtc
Message
listen addr=:8555
```

How do you know if an individual camera is streaming using webrtc? Does the /#birdseye page support streaming certain cameras over webrtc?

**Comment by NickM-27** [comment]
By default MSE will be used, WebRTC will always be used when using two way talk or if MSE fails to stream
